lib/sample_uploader/utils/samples_content_error.py

```diff
@@ -0,0 +1,16 @@
+class SampleContentError(Exception):
+    def __init__(self, message, sample_name=None, node=None, key=None, row=None, column=None):            
```

Comment:
could you add a short description of what each field is and include the expected type

Reply:
on it